pkg/cmd/util/output/backup_storage_location_printer.go

```diff
@@ -55,13 +56,19 @@ func printBackupStorageLocation(location *v1.BackupStorageLocation) []metav1.Tab
 
 	accessMode := location.Spec.AccessMode
 	if accessMode == "" {
-		accessMode = v1.BackupStorageLocationAccessModeReadWrite
+		accessMode = velerov1api.BackupStorageLocationAccessModeReadWrite
+	}
+
+	status := location.Status.Phase
+	if status == "" {
+		status = velerov1api.BackupStorageLocationPhaseAvailable
```

Comment:
I'd display this as `<Unknown>` or something along those lines, since we don't yet know if it's available.

Even if we define a third phase of `Unknown`, new CRs could still have an empty phase (until we add defaulting to our CRD specs), so I do think it makes sense to check for and handle an empty status.phase field here.

Reply:
Yes, make sense. Looks like this now:

![image](https://user-images.githubusercontent.com/16508/83081111-c01b9c80-a034-11ea-98d8-b5f427aa59d0.png)